apps/browser/qml/pages/components/BookmarkItem.qml

```diff
@@ -101,6 +94,16 @@ ListItem {
                 text: qsTrId("sailfish_browser-me-copy-to-clipboard")
                 onClicked: Clipboard.text = model.url
             }
+            MenuItem {
+                //% "Add to App Grid"
```

Comment:
No need to re-define. You can leave a normal comment that referring to the FavoriteContextMenu.